superdesk/metadata/item.py

```diff
@@ -108,17 +108,14 @@
     # Copyright Information
     'usageterms': {
         'type': 'string',
-        'mapping': not_analyzed,
         'nullable': True,
     },
     'copyrightnotice': {
         'type': 'string',
```

Comment:
What sort of data will be there in `copyrightnotice`? 
Are we going to search on `copyrightnotice`?